vscode/src/circuit.ts

```diff
@@ -315,7 +315,7 @@ function errorsToHtml(
 
     const location = documentHtml(document, diag.range);
     const message = escapeHtml(`(${diag.code}) ${diag.message}`).replace(
```

Comment:
replaceAll has also been common for about 4 years to address this common mistake <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll>